apps/web/src/hooks/useVersions.ts

```diff
@@ -16,6 +17,10 @@ export function useVersions() {
     queryKey: ['versions', selectedProject],
     queryFn,
     select: (data) =>
-      data.filter(tag => tag.includes(`@zk-kit/${selectedProject}-artifacts@`)).map(tag => tag.split('@')[2]),
+      data
+        .filter(tag => tag.includes(`@zk-kit/${selectedProject}-artifacts@`))
+        .map(tag => tag.split('@')[2])
+        .filter(version => version !== 'latest')
+        .sort(rcompare),
```

Comment:
if we had a huge number of versions, which is not the case, I'd probably refactor this in one loop for efficiency. I prefer having something easier to read for now